segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/admin/commands/State.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.server.host.admin.commands;
+
+import io.pravega.common.concurrent.ExecutorServiceHelpers;
+import io.pravega.segmentstore.server.store.ServiceBuilderConfig;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.concurrent.ScheduledExecutorService;
+import lombok.Getter;
+
+/**
+ * Keeps state between commands.
+ */
+public class State implements AutoCloseable {
```

Comment:
Can you come up with a better name for this?

Reply:
Renamed to `AdminCommandState`